flybywiresim/aircraft
# fix(a380x/mfd): missed approach perf page inputs

Fixes #[issue_no]

## Summary of Changes

Changes MFD GA approach page altitude inputs so they affect the correct performance data in the FMS.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

1) Open PERF TO page on the MFD without any airport pair inserted in INIT A. THR RED & ACCEL fields should be empty.
2) Insert an airport city pair of two different airports on the INIT page and verify the PERF TO Page again. The inserted thrust reduction, engine out & acceleration fields should go in line with the EFB settings (airfact options/pin programs).
3) Verify the heights on the GA PERF page follow the same rule as above but based on the landing airport elevation.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
